models/token.js

```diff
@@ -17,7 +17,7 @@ const MODEL = {
 
     hash: Joi
         .string()
-        .base64()
+        .regex(/[a-zA-Z0-9_-]+~/)
```

Comment:
I feel like it might be simpler just to define a function in the one file that deals with string encodings (models/generateToken.js) that runs the replace, rather than bringing in a dependency. All that needs to be done is run `string.replace` a few times. I could be wrong on that though.